### PR TITLE
PHP8 compatibility

### PR DESCRIPTION
**Pull request by Geekimo**
PHP8 compatibility and Sf5 test

### CODE REVIEW

**Comment by dontub** [comment]
Thanks for the PR!  I currently see two issues:
1. There are some jobs failing: https://travis-ci.org/github/shivas/versioning-bundle/builds/754022997
2. nikolaposa/version 3.2 is not marked as PHP 8 compatible.

File: .travis.yml [hunk 1]
```diff
@@ -38,6 +39,10 @@ matrix:
         - php: 7.3
           env: SYMFONY_VERSION="^4"
 
+          # Test with Symfony 5
+        - php: 7.3
+          env: SYMFONY_VERSION="^5"
```
Comment (dontub) [inline .travis.yml:44]: This is not required because by default the latest Sf 5 version is automatically chosen by composer which already happens above.

Comment (Geekimo) [reply .travis.yml:44]: Yep, i'm not used to travis...
I'll correct this.

**Comment by Geekimo** [comment]
Regarding nikolaposa's work, the latest version is php8 compatible, i'll update the composer.json

**Comment by dontub** [comment]
In #73 updating to nikolaposa/version 4 is WIP. I'd suggest to wait for this and afterwards add PHP 8 compatibility.

**Comment by Geekimo** [comment]
Hi @dontub ,
I was talking with this issue with my CTO this morning, as we are trying to go PHP8 for all our future developments, and many librairies, despite having php8 support in master branch doesn't have tagged it yet.

Thanks for the feedback tho.

**Comment by xvilo** [comment]
It would be great if this could already be merged as it will allow possibly faster compatibility

**Comment by Geekimo** [comment]
#73 is way more advanced than mine.

**Pull request closed without merging**